src/app/proposals/draft/utils/github.ts

```diff
@@ -0,0 +1,351 @@
+"use server";
```

Comment:
A lot of todos, comments, and console.logs. Let's nuke the code we are not using.

Reply:
I think there are actually mistakes in this file that I need to go back and fix. Thanks for calling this out.